frontend/src/metabase/dashboard/components/DashboardGrid.tsx

```diff
@@ -292,7 +292,7 @@ class DashboardGrid extends Component<DashboardGridProps, DashboardGridState> {
 
   getLayoutForDashCard = (dashcard: BaseDashboardCard) => {
     const visualization = getVisualizationRaw([
-      { card: (dashcard as DashboardCard).card } as SingleSeries,
+      { card: (dashcard as QuestionDashboardCard).card } as SingleSeries,
```

Comment:
Can we avoid the first type cast by using a type-guard?

Alternatively we should be able to use `DashboardCard` instead of `BaseDashboardCard` throughout this file.

Reply:
Fixed in [`61c17a7`](https://github.com/metabase/metabase/pull/38444/commits/61c17a7e716a2ae262f6daa87b753eb9aa2629c3)